docs/source/utils/vmimage.rst

```diff
@@ -97,7 +97,7 @@ with::
     class MyTest(Test):
 
         def setUp(self):
-            vmimage.IMAGE_PROVIDERS.append(MyProvider)
+            vmimage.IMAGE_PROVIDERS.add(MyProvider)
```

Comment:
OK, you changed the append used for the list to the add, appropriate for the set. Nice